nyxx/lib/src/core/channel/ThreadChannel.dart

```diff
@@ -0,0 +1,50 @@
+part of nyxx;
+
+class ThreadChannel extends MinimalGuildChannel implements TextChannel {
+
+  Timer? _typing;
+
+  ThreadChannel._new(INyxx client, Map<String, dynamic> raw, [Snowflake? guildId]) : super._new(client, raw) {}

```

Comment:
No need for braces. just semicolon at the end

Reply:
Will be updated soon with more features.